client/question/services/ConversationManagerService.js

```diff
@@ -78,9 +108,43 @@ tie.factory('SolutionHandlerService', [
             return CodeRunnerDispatcherService.runCodeAsync(
               language, codeSubmission.getPreprocessedCode()
             ).then(function(preprocessedCodeEvalResult) {
-              feedback = FeedbackGeneratorService.getFeedback(
-                tasks, preprocessedCodeEvalResult,
-                codeSubmission.getRawCodeLineIndexes());
+              var feedbackDetails = computeFeedbackDetails(
+                preprocessedCodeEvalResult);
+
+              // TODO(sll): Remove this case.
```

Comment:
Please add a reason why we should remove this case, just in case you get hit by a bus or something and we need to crack the code. :P

Reply:
Done :)